laalaa/apps/advisers/importer.py

```diff
@@ -174,12 +174,149 @@ def drop_csv_table(self, csv_filename):
 
     def translate_data(self):
         cursor = connection.cursor()
+
+        cursor.execute("DROP FUNCTION IF EXISTS count_office_relations(integer);")
+
+        cursor.execute("""
+            CREATE OR REPLACE FUNCTION count_office_relations(
```

Comment:
if the return type changes it needs to be dropped first, so incase the function gets changed.